app/components/nft-web-view/nft-web-view.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import { WebView, WebViewNavigation, WebViewProps } from "react-native-webview"
+
+import { COMMON_API_CONFIG } from "../../services/api/api-config"
+import { Linking } from "react-native"
+import { OnShouldStartLoadWithRequest } from "react-native-webview/lib/WebViewTypes"
+
+export function NFTWebView(props: WebViewProps) {
+
+  const handleShouldStartLoadWithRequest: OnShouldStartLoadWithRequest = ({ url }: WebViewNavigation) => {
+    if (/\.(pdf|epub)/i.test(url)) {
```

Comment:
url is not reliable, would be best if we have mimetype, but not sure what does WebViewNavigation provide